tools.js

```diff
@@ -130,20 +173,19 @@ module.exports = (function () {
 			});
 		}
 	}
+
 	Tools.loadMods = function () {
```

Comment:
On second thought, this might be contraproducent.
Ok, will revert this part. I have to rebase anyway...